src/v/kafka/protocol/schemata/describe_producers_request.json

```diff
@@ -0,0 +1,30 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+{
+    "apiKey": 61,
+    "type": "request",
+    "name": "DescribeProducersRequest",
+    "validVersions": "0",
+    "flexibleVersions": "0+",
+    "fields": [
+        { "name": "Topics", "type": "[]TopicRequest", "versions": "0+", "fields": [
+          { "name": "Name", "type": "string", "versions": "0+", "entityType": "topicName",
+            "about": "The topic name." },
+          { "name": "PartitionIndexes", "type": "[]int32", "versions": "0+",
```

Comment:
please add mapping of partition index type to `model::partition_id` in generator.py

Reply:
It is from kafka. 
Should I use another type fro it?

Reply:
Here we still use []int32 but when we generate classes if will be replaced with `model::partition_id`. Grep `partition_id` in the `generator.py` for an example

Reply:
It is not the same as we already have

Here we do not have array of structures, where one of the filed is int32 

We have field which is array. So I am not sure we can define that it should be array of model::partition_id